spec/support/strict_warnings.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module StrictWarnings
```

Comment:
It'd be good to add some module-level documentation as well - basically an explanation of the purpose of this module.